joint_ids:
- 2
- 3
- 8
- 9
- 6
- 5
- 1
- 7
- 4
- 0
joint_names:
- j11
- j12
- j13
- j14
- j21
- j22
- j23
- j31
- j32
- j33
